pkg/azurefile/version_test.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package azurefile
+
+import (
+	"testing"
+
+	"sigs.k8s.io/yaml"
+)
+
+func TestGetVersion(t *testing.T) {
+	get := VersionInfo{
+		DriverName:    "DriverName",
+		DriverVersion: "Driver Version",
+		GitCommit:     "Git Commit",
+		BuildDate:     "Build Date",
+		GoVersion:     "Go Version",
+		Compiler:      "Compiler",
+		Platform:      "Platform",
+	}
+	want := VersionInfo{
+		DriverName:    "DriverName",
+		DriverVersion: "Driver Version",
+		GitCommit:     "Git Commit",
+		BuildDate:     "Build Date",
+		GoVersion:     "Go Version",
+		Compiler:      "Compiler",
+		Platform:      "Platform",
+	}
+	if get == want {
+		t.Log("Got as expected")
+	} else {
+		t.Errorf("Unexpected Versions: Please check the requirements")
+	}
+
+}
+
+func TestGetVersionYAML(t *testing.T) {
+	get := GetVersion()
```

Comment:
Should test against `GetVersionYAML` func

Reply:
Changes done